src/sentry/snuba/discover.py

```diff
@@ -105,106 +113,155 @@ def is_real_column(col):
     return True
 
 
-def format_time(data, start, end, rollup, orderby):
-    rv = []
-    start = int(to_naive_timestamp(naiveify_datetime(start)) / rollup) * rollup
-    end = (int(to_naive_timestamp(naiveify_datetime(end)) / rollup) * rollup) + rollup
-    data_by_time = {}
+def format_time(
+    data: SnubaData, start_param: datetime, end_param: datetime, rollup: int, orderby: list[str]
```

Comment:
Needed to rename the start & end params since their type changes later on 121

Reply:
:+1: not used anywhere except `src/sentry/snuba/metrics_performance.py` which just passes params